datastore/postgres/types/version.go

```diff
@@ -0,0 +1,155 @@
+package types
```

Comment:
This is copied directly from https://github.com/hdonnay/claircore/tree/hack/db-updates and allows `claircore.Version`s to `Scan()` natively (which is awesome). There are other ScanPlans/EncodePlans in that branch that would probably be nice to add at some point, but currently the version code is required to make versions function correctly with v5.